src/style.css

```diff
@@ -0,0 +1,120 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+input,
+button {
+  font-family: inherit;
+  outline: none;
+}
+
+img {
+  object-fit: contain;
+  width: 100%;
+  height: 100%;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.container {
+  padding: 0 8px;
+}
+
+.form-searchbar {
+  position: relative;
+  height: 70px;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  overflow: hidden;
+  transition: all 0.3s ease;
+  z-index: 3;
+}
+
+.form-searchbar--small {
+  height: 42px;
+}
+
+.searchbar__button {
+  position: absolute;
+  top: 50%;
+  left: 26px;
+  transform: translate(0, -53%);
+  background: transparent;
+  width: 19px;
+  height: 19px;
+  cursor: pointer;
+}
+
+.searchbar__button--small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+}
+
+.searchbar__input {
+  width: 100%;
+  height: 100%;
+  font-family: Avenir, serif;
```

Comment:
Change the font-family to 'Avenir, sans-serif;' to use a sans-serif fallback font.